src/arch/io/network.cc

```diff
@@ -621,6 +621,430 @@ void linux_tcp_conn_t::on_event(int /* events */) {
     event_watcher->stop_watching_for_errors();
 }
 
+/* tls_conn_wrapper_t wraps a TLS connection and safely releases it in
+exceptional cases if it is not reclaimed before execution leaves its scope. */
+class tls_conn_wrapper_t {
+public:
+    tls_conn_wrapper_t(SSL *_conn) {
```

Comment:
I don't mind at all. I should be able to make fixes like these very quickly. I know coding style consistency is very important for readability and maintainability. Sorry, I didn't even think to look for a style guide, but I figured the project maintainers would point these things out :+1: